controllers/sdkutil/clientwrapper/sagemaker_client.go

```diff
@@ -35,6 +36,9 @@ const (
 	StopTrainingJob404Code              = "ValidationException"
 	StopTrainingJob404MessagePrefix     = "Requested resource not found"
 
+	DescribeProcessingJob404Code          = "ValidationException"
```

Comment:
the name specifically has 404 to indicate that SM throws this error when it should have been 404

yes we do have different constant maybe because not all APIs show same behaviour